Task_7.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"sort"
+	"time"
+)
+
+func (p People) Len() int {
+	return len(p)
+}
+
+func (p People) Less(i, j int) bool {
+
+	if p[i].birthDay.After(p[j].birthDay) {
+		return true
+	}
+
+	if p[i].birthDay.Equal(p[j].birthDay) && p[i].firstName < p[j].firstName {
+		return true
+	}
+	return false
+}
+
+func errCheck(err error){
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+}
+
+func (p People) Swap(i, j int) {
+	p[i], p[j] = p[j], p[i]
+}
+
+type Person struct {
+	firstName string
+	lastName  string
+	birthDay  time.Time
+}
+
+type People []Person
+
+func main() {
+
+	dateFirst, err := time.Parse("2006-Jan-02", "2005-Aug-10")
+	errCheck(err)
+	dateSecond, err := time.Parse("2006-Jan-02", "2003-Aug-05")
+	errCheck(err)
+	dateThird, err := time.Parse("2006-Jan-02", "2012-Nov-21")
+	errCheck(err)
+	dateFourth, err := time.Parse("2006-Jan-02", "1999-Dec-12")
+	errCheck(err)
+	dateFifth, err := time.Parse("2006-Jan-02", "2001-Jan-08")
+	errCheck(err)
+
+	p := People{
+		{"Artiom", "Ivanov", dateFirst},
```

Comment:
```suggestion
            {firstName: "Artiom", lastName:"Ivanov", birthDay: dateFirst},
```
Just a safe code (e.g. field order in struct _could_ be changed)
And this is a bit more readable.
Not a blocker though.